frontend/src/components/layouts/SearchBar.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <div v-if="visible" class="w-screen h-screen bg-gray-500 bg-opacity-40 backdrop-blur-md">
+      <div @click="closeModal" class="absolute inset-0"></div>
+      <div class="relative z-50 bg-white shadow-lg w-full p-6">
+        <div class="flex items-center justify-between border-b pb-4">
+          <input 
+            type="text" 
+            class="w-full p-2 text-lg border-transparent focus:border-none focus:ring-0 focus:outline-none font-semibold" 
+            placeholder="Buscar..." 
```

Comment:
Make sure in translate implementation change this for `Search . . .` and `Buscar . . .`

Reply:
I'm gonna attend this point in the next Pull Requests for implement translate in all frontend and it's not in only component in specify.